packages/jaspr_tailwind/lib/builder.dart

```diff
@@ -46,15 +44,15 @@ class TailwindBuilder implements Builder {
         scratchSpace.fileFor(buildStep.inputId).path,
         '--output',
         scratchSpace.fileFor(outputId).path.toPosix(),
-        '--content',
-        p.join(Directory.current.path, '{lib,web}', '**', '*.dart').toPosix(true),
         if (options.config.containsKey('tailwindcss')) options.config['tailwindcss'],
         if (hasCustomConfig) ...[
           '--config',
           p.join(Directory.current.path, 'tailwind.config.js').toPosix(),
+        ] else ...[
+          '--content',
+          p.join(Directory.current.path, '{lib,web}', '**', '*.dart').toPosix(true),
         ],
       ],
-      workingDirectory: root,
```

Comment:
This shouldn't be removed. If you think there is a bug or smth let me know.

Reply:
If Tailwind is not run in the project directory, the `content` paths in `tailwind.config.js` are not detected correctly and CSS is not built properly. Originally, I thought it was common to run the build from the project location when using Tailwind, so may I ask why you set the `workingDirectory` that way?

Reply:
This is because build_runner creates a temporary directory where it copies all the project files to and then operates on. So the command should also be run in that temporary directory and not the project dir itself.

If you have a custom config that causes problems can you send me it then I can try to reproduce and see where the problem is.

Reply:
As in the example I added to the documentation, even if you just add `content: ['./{lib,web}/**/*.dart']`, Tailwind won't be able to find the content location properly because the working directory is not the project root directory.

Reply:
Also, after checking with `print`, I found that the temporary folder created by `build_runner` and the path in the `root` variable in the original code before I modified it seem to be different. However, there has been no problem in using it so far. Below is the result confirmed on my computer:

`print(scratchSpace.fileFor(buildStep.inputId).path)`: /private/var/folders/tr/n368kyf54klb2cm37fw2dy_w0000gn/T/scratch_spaceTVDt3X/web/styles.tw.css
`print(root)`: /Users/hyunwoopark/.pub-cache/hosted/pub.dev/jaspr_tailwind-0.2.0

Reply:
Ah thanks for debugging this, then the root var is wrong and needs to be set to the scratch space dir.

Reply:
Then, if `tailwindcss` is executed from the scratch space dir as you said, all files belonging to the `content`, such as files under a `lib` folder, should be copied to the same relative path based on the scratch space dir for the `content`. However, `scratchSpace.ensureAssets` copies files under a `lib` folder to a `packages` folder corresponding to its package, not directly under the scratch space dir. Therefore, it is difficult to detect all files belonging to the `content` and copy them to the scratch space dir as they are to build Tailwind, and I don't know why it is necessary to do that. I am curious to know your thoughts on this. Thanks.

Reply:
Thanks for diving into this, you changed my mind. It indeed seems to make more sense to run this in the normal project dir. Just the output should still be written to the scratch space.

I will resolve this and do some more testing.